web/src/cypress/types/alert.d.ts

```diff
@@ -0,0 +1,40 @@
+declare namespace Cypress {
+  interface Chainable {
+    createAlert: typeof createAlert
+    createManyAlerts: typeof createManyAlerts
+    closeAlert: typeof closeAlert
+    createAlertLogs: typeof createAlertLogs
+  }
+}
+
+interface Alert {
+  number: number
+  id: number
```

Comment:
Are we still using the old API for alert calls? New schema should be `id` and `alertID`

Reply:
Sounds good